doc/nrf/releases/release-notes-1.8.0.rst

```diff
@@ -691,15 +691,13 @@ DFU target
 MCUboot
 =======
 
-The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``680ed07``, plus some |NCS| specific additions.
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``70bfbd2``, plus some |NCS| specific additions.
 
 The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
 
 The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
 
-* The value of the :kconfig:`CONFIG_PM_PARTITION_SIZE_MCUBOOT_SECONDARY` Kconfig option does not have to be specified manually as it automatically shares the value with the primary partition.
-
-* Reduced the number of downstream patches to the :file:`ncs/bootloader/mcuboot/boot/zephyr/Kconfig` file.
+* No entries for this section.
```

Comment:
@nvlsianpu , what summary should we add here?

Reply:
- fixed support for low power in zephyr's single thread mode, see MCUBOOT_CPU_IDLE() macro
- switched usb cdc-acm serial recovery I/O device configuration from Kconfig to DT with compatible = "zephyr,cdc-acm-uart"
- switched uart serial recovery I/O device configuration from Kconfig to DT using zephyr's zephyr,console property of the chosen node.
- fixed deadlock on cryptolib selectors in Kconfig
- Fixed serial recovery skipping on nrf5340
- Added cleanup of uarte devices before chainload the application which allows the application to initialize them correctly (this fixes for instance possible missing output of the application's log issue)